integration/volume_test.go

```diff
@@ -75,3 +75,23 @@ func (s *CliSuite) TestNamedVolume(c *C) {
 	c.Assert(cn.Mounts[0].Name, DeepEquals, "vol")
 	c.Assert(cn.Mounts[0].Destination, DeepEquals, "/path")
 }
+
+func (s *CliSuite) TestV2Volume(c *C) {
```

Comment:
I think an integration test for external volumes would be pretty useful as well.